infrateam_first_responder.md

```diff
@@ -41,23 +41,25 @@ The idiosyncratic name of the role is intentional. It is not the duty of the fir
+The first responder needs to make sure that all codebases needing updates have updates merged and deployed. Note that some projects may need to have PRs created by hand where automatic creation may have failed. It is helpful to post updates in the `#dlss-infrastructure` Slack channel to make sure the team is aware of this work, in case anyone is working in related codebases or looking to deploy changes.
 
 ##### Merge 'em
-WIP script to automatically merge all dependency update PRs is currently in its own PR in access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/104/files -- you can switch to the branch locally and use this script.  From the comments at the top, you will need a github access token.  Instructions are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
+
+Run the `merge-all` script to automatically merge all dependency update PRs: https://github.com/sul-dlss/access-update-scripts/blob/master/merge-all.rb. Note that this script will only work with Ruby 2.6 or greater.  See the comments at the top for how to run and note you will need a github access token if you haven't previously created one.  Instructions for creating a token are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line   Save your token somewhere secure for re-use since you won't be able to view it in the Github interface again.
 
 ##### Deploy 'em
-Use the `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below) via capistrano to deployed environments -- see here:  https://github.com/sul-dlss-labs/sdr-deploy.  
 
-There are applications that need to be deployed separately (i.e., not using `sdr-deploy`), currently (6/9/2020): sinopia apps, dlme-transform, and rialto-webapp are deployed using Terraform. Also note `sdr-deploy` is not a good tool for deploying the hydra_etd application to the `uat` environment nor for deploying the sul-pub application to its environments beyond `stage` and `prod`: https://github.com/sul-dlss/sul_pub/tree/master/config/deploy.
+Use the `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below) via capistrano to deployed environments: https://github.com/sul-dlss/sdr-deploy
+
+**NOTE**: There are applications that need to be deployed separately (i.e., not using `sdr-deploy`), currently (as of August 2020): sinopia apps, dlme-transform, and rialto-webapp are deployed using Terraform. Also note `sdr-deploy` is not a good tool for deploying the hydra_etd application to the `uat` environment nor for deploying the sul-pub application to its environments beyond `stage` and `prod`: https://github.com/sul-dlss/sul_pub/tree/master/config/deploy. And these need to be done every week.
```

Comment:
if this were more skimmable (think bullet points, maybe?) then I think it would be more effective -- I see a paragraph that starts "NOTE" and I am inclined to skip it.  If nothing else, at least change it to "IMPORTANT" and bold the repo names.

Reply:
@ndushay https://github.com/sul-dlss/DeveloperPlaybook/pull/132/commits/e4c3eef37a7646c90c59e97fbe96a5d06f46df54 

thanks! re-review?